documentation/developer_setup.md

```diff
@@ -0,0 +1,126 @@
+# Developer Set Up
+
+* [Installing Go](#installing-go)
+* [Create a Client](#create-a-client)
+	* [Documenting Code](#documenting-code)
+    * [Constructors](#constructors)
+	* [Defining Methods](#defining-methods)
+* [Write Tests](#write-tests)
+
+## Installing Go
+
+The Azure-sdk-for-go team supports Go versions 1.14 and greater, with our CI pipelines testing against versions 1.14 and 1.15 on both Windows and Linux virtual machines. If you do not already have Go installed, refer to this [workspace setup][workspace_setup] article for a more in depth tutorial on setting up your Go environment. After installing Go and configuring your workspace, fork the `azure-sdk-for-go` repository and clone it to a directory that looks like: `<GO HOME>/src/github.com/Azure/azure-sdk-for-go`.
```

Comment:
Instead of mentioning the versions, call out that we support the N and N-1 releases of Go.  Then point to the pipeline yaml file which calls out the versions
https://github.com/Azure/azure-sdk-for-go/blob/main/eng/pipelines/templates/jobs/archetype-sdk-client.yml